generators/angular/templates/src/main/webapp/app/app.config.ts.ejs

```diff
@@ -36,7 +36,19 @@ import routes from './app.routes';
 import { NgbDateDayjsAdapter } from './config/datepicker-adapter';
 import { AppPageTitleStrategy } from './app-page-title-strategy';
 
-const routerFeatures: Array<RouterFeatures> = [withComponentInputBinding()];
+const routerFeatures: Array<RouterFeatures> = [withComponentInputBinding(),
+  withNavigationErrorHandler((e: NavigationError) => {
+    const router = inject(Router);
+    if (e.error.status === 403) {
+      router.navigate(['/accessdenied'])
```

Comment:
IntelliJ doesn't like that there's no `await` or `.then()` here, but it works, so I'm OK with not having it. 

<img width="886" alt="Screenshot 2023-12-10 at 9 35 30 AM" src="https://github.com/jhipster/generator-jhipster/assets/17892/810aed26-080f-4a46-99df-5d3c537be2f1">

It also says the `httpInterceptorProviders` import can be shorted to:

```ts
import { httpInterceptorProviders } from './core/interceptor';
```

Reply:
It also says the `httpInterceptorProviders` import can be shorted to:

```ts
import { httpInterceptorProviders } from './core/interceptor';
```